### PR TITLE
update man pages & conf file

### PR DESCRIPTION
**Pull request by gravitypriest**

### CODE REVIEW

File: docs/insights-client.8 [hunk 1]
```diff
@@ -40,6 +40,16 @@ Print archive to stdout; sets --silent and --no-upload.
 Specifies the compression algorithm to use. Choices are gz, bz2, xz, and none. Defaults to gz.
 .IP "--offline"
 Collect locally only, do not connect to Insights and do not upload.
+.IP "--analyze-container"
```
Comment (brantleyr) [inline docs/insights-client.8]: I would just note that in the core client code (config.py) that --analyze-container, --analyze-image-id, --analyze-file and --analyze-mountpoint have been suppressed. So they will show up in the man pages but not in --help.

**Comment by gravitypriest** [comment]
Ya that's another change I gotta make

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Comment by YakovLittle** [comment]
test this please

**Pull request merged**